### ISSUE TITLE
Notifications: Reply keyboard should be the default one

### ISSUE THREAD

**Issue by jleandroperez** [wordpress-mobile/WordPress-iOS#2835]
Recently ReplyTextView's keyboard type was switched to Twitter, due to the Mentions enhancements.

As discussed, this may prove confusing for some users, since the intro key is not immediately accessible.

Let's revert back to the default keyboard.
